### ISSUE TITLE
osbuild-built Fedora Workstation live image is ~300 MB bigger than livemedia-creator-built one

### ISSUE THREAD

**Issue by AdamWill** [osbuild/images#461]
The Fedora Workstation live image built by osbuild is ~300 MB bigger than the one built by livemedia-creator, in any given compose. e.g. at https://kojipkgs.fedoraproject.org/compose/rawhide/Fedora-Rawhide-20240220.n.0/compose/Workstation/x86_64/iso/ , the osbuild image is 2595389440 bytes, the 'normal' (livemedia-creator built) image is 2266372096 bytes.

Not sure why this is, but it seems worth investigating at least.

**Comment by supakeen** [comment]
@bcl I know lorax does some cleaning up for the live images but I forgot what the lorax template was/where I could find it. Do you remember where it is?

@achilleas-k We'd likely want to implement this in [osbuild](https://github.com/osbuild/osbuild) as a separate stage however do we want separate stages per step or a single stage that would take a variety of cleanup options?

**Comment by AdamWill** [comment]
lorax cleans up installer environments, but not live ones, AFAIK. https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/runtime-cleanup.tmpl is the cleanup template for the installer environment, but I see no equivalent in https://github.com/weldr/lorax/tree/master/share/templates.d/99-generic/live .

**Comment by supakeen** [comment]
@AdamWill Ok. So the source of truth for any cleanups performed by the live installers would be fully in the kickstarts?

**Comment by bcl** [comment]
Right, for the live if you want to clean things up you have to do it in the kickstart, either manually or using the `image-minimizer` script like this - https://github.com/weldr/lorax/blob/master/docs/fedora-minimized.ks#L99
Note, I'm not sure how much use there is of that method, I don't think I've ever received any bug reports for it.

Here's a short description of how lmc works - https://weldr.io/lorax/livemedia-creator.html#how-iso-creation-works
Basically, it's the output from Anaconda wrapped in an iso.

Official fedora kickstarts are here - https://pagure.io/fedora-kickstarts/tree/main
